site/src/atomics/aspect-ratio.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Aspect ratio
+description: Aspect ratio related atomic css classes for the Atlas Design System
+template: standard
+classType: Atomics
+classPrefixes:
+  - aspect-ratio
+---
+
+# Aspect Ratio Atomics
+
+Ratio atomics are used to add a fluid aspect ratio to an element. The element will scale to fill 100% width of the container while matching the height to the defined aspect ratio.
+
+| cssproperty  | ratio                                             | screensize |
+| ------------ | ------------------------------------------------- | ---------- |
+| aspect-ratio | `1-1`, `2-1`, `3-2`, `4-3`, `5-3`, `9-16`, `16-9` | N/A        |
```

Comment:
4-3 and 5-3 definitely still needed?

Reply:
Removed 3-2 and 5-3. Left 4-3 because it's a standard video and monitor ratio.